reV/handlers/outputs.py

```diff
@@ -210,772 +141,25 @@ def package(self):
         return self.h5.attrs['package']
```

Comment:
Why do we need this package property copied in the reV handler?

Reply:
we don't lol. removed.